app/react/UI/Icon/library.js

```diff
@@ -175,10 +176,13 @@ const icons = {
   faUsers,
   faUserTimes,
   faHandPaper,
+  saveAndNext,
 };
 
 export const loadIcons = () => {
   library.add(...Object.keys(icons).map(key => icons[key]));
 };
 
+dom.watch();
```

Comment:
comment to doc?

Reply:
Why is this  required and why was it not needed before? Thanks.

Reply:
Looks like it isn't required. It was in the instructions to add a custom icon, but those weren't react specific instructions. I'm guessing that the component already calls this.